### PR TITLE
DPPA-436: Document loading and segmenting mibitiff in cell profiler and Qupath

### PR DESCRIPTION
**Pull request by jaytarolli**

### CODE REVIEW

File: scripts/qupath_export_label_image.groovy [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+}
+
+g2d.dispose()
+
+new ImagePlus('Labels', labelImage).show()
+
+def name = getProjectEntry().getImageName()
+def path = buildFilePath(PROJECT_BASE_DIR)
+def fileoutput = new File(path, 'labels.png')
```
Comment (rfinck) [inline scripts/qupath_export_label_image.groovy:44]: This saves the file as 'labels.png' in the top level of the project directory, but if I have multiple images in my project, then it'll overwrite each one if I run it on each.

File: scripts/qupath_export_label_image.groovy [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ImageIO.write(labelImage, 'png', fileoutput)
```
Comment (rfinck) [inline scripts/qupath_export_label_image.groovy:45]: Please add a newline at the end.

**Review by rfinck — COMMENTED** [review]
I would suggest putting this in a `qupath` or `qupath_scripts` or `scripts/qupath` folder rather than simply `scripts`.

File: scripts/qupath_export_label_image.groovy (hunk 1 above)
Comment (jaytarolli) [reply scripts/qupath_export_label_image.groovy:44]: I can append "labels" to the end of the name as supplied by QuPath, but it is formatted "[original file name].tiff - [original file name].tiff SIMS - labels.png". That seems to be a limitation of how QuPath builds an image name and I don't immediately see a way around it.

Comment (rfinck) [reply scripts/qupath_export_label_image.groovy]: I was going to suggest some string manipulation such as 
```
def imageName = getProjectEntry().getImageName()
def originalName = imageName.split('.tiff - ')[0]
```
but I believe the `getOriginalImageName` method is actually what you want:
```
def imageName = getProjectEntry().getOriginalImageName()
```

**Comment by rfinck** [comment]
@jaytarolli What's the state of this? Is it awaiting another review?

**Comment by jaytarolli** [comment]
@rfinck I believe I was waiting on your input regarding my last comment about the output file name.

**Review by rfinck — APPROVED** [review]
Sorry for the super long delay, feel free to remind me next time!

**Pull request merged**